cmd/frontend/internal/pkg/search/query/types/check.go

```diff
@@ -44,16 +44,29 @@ func (c *Config) Check(query *syntax.Query) (*Query, error) {
 		Syntax: query,
 		Fields: map[string][]*Value{},
 	}
+
+	fixedExprs := []string{}
+	errCount := 0
+
 	for _, expr := range query.Expr {
+		fx := expr.String()
 		field, fieldType, value, err := c.checkExpr(expr)
 		if err != nil {
-			return nil, err
+			errCount++
+			fx = `"` + fx + `"`
```

Comment:
Let's make this more specific:

```suggestion
if err != nil {
	if expr.ValueType != syntax.TokenPattern {
		return err
	}
	errCount++
	fx = `"` + fx + `"`
}
```